src/python/turicreate/toolkits/sound_classifier/_mx_sound_classifier.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2019 Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can
+# be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+
+
+import mxnet as mx
+import time
+import turicreate as _tc
+
+
+class SoundClassifierMXNETModel():
+
+
+	def __init__(self, feature_extractor, num_labels, custom_layer_sizes, verbose, ctx):
+		from .sound_classifier import SoundClassifier
+		from .._mxnet import _mxnet_utils
+		
+		self.verbose = verbose
+		self.custom_NN = SoundClassifier._build_custom_neural_network(feature_extractor.output_length, num_labels, custom_layer_sizes)
```

Comment:
`_build_custom_neural_network` should be moved too, since it includes MXNet code

Reply:
Moved to where? It is moved from the main file to the SC MXNet class

Reply:
Ohhh do you mean move it from the CustomModel class over to this one?

Reply:
Yup, the long-term goal is to get all of the MXNet code out of the CustomModel class. For the first PR, my current guess would be to move everything except for the data iterator and metric

Reply:
(Since the data iterator and metric are compatible with a TF code path. The current implementation of `_build_custom_neural_network` is not compatible/usable by the TF code path

Reply:
Ahhh ok. Now I understand better what our plan is